src/executables/util_cs.c

```diff
@@ -3212,6 +3215,10 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
 
   check_replica_info = (HA_GET_MODE () == HA_MODE_REPLICA);
   pageid = utility_get_option_bigint_value (arg_map, APPLYINFO_PAGE_S);
+  if ((pageid != -1) && (log_path == NULL))
+    {
+      goto print_applyinfo_usage;
+    }
```

Comment:
```suggestion
   pageid = utility_get_option_bigint_value (arg_map, APPLYINFO_PAGE_S);
  if ((pageid != NULL_LOG_PAGEID) && (log_path == NULL))
    {
      goto print_applyinfo_usage;
    }
    
  check_replica_info = (HA_GET_MODE () == HA_MODE_REPLICA);
 
```

how about change the sequence ? 
because pageid is related to log_path, so it will be more readable when change the sequence. 

Does the pageid default to -1 if the user doesn't use the -p option? 
where the validation code for pageid? 
`la_log_page_check` just check if pageid is larger than 1. 
If the user sets the pageid less than or equal to 1 ( 1, 0 , -1 ,...), no error is printed.

Reply:
> how about change the sequence ?
> because pageid is related to log_path, so it will be more readable when change the sequence.

Answer: Good idea. Thank you. I will change my PR.

> Does the pageid default to -1 if the user doesn't use the -p option?

Answer: Yes. and if I use p option as `-p -1`, pageid is also -1. (invalid case)

> where the validation code for pageid?
> la_log_page_check just check if pageid is larger than 1.
> If the user sets the pageid less than or equal to 1 ( 1, 0 , -1 ,...), no error is printed.

Answer: No validation code except `la_log_page_check`.
This problem is complicated to handle the bug. because when I have the meeting with @hornetmj, we concluded this issue is the out of range in this issue. The validation seems be related with internal log page. 
[default page is 0](https://www.cubrid.org/manual/en/11.0/ha.html?highlight=applyinfo#cmdoption-applyinfo-p) as shown on manual, page no.0 means active page.

Reply:
@joohok 
Please Ignore [above comment](https://github.com/CUBRID/cubrid/pull/3374#discussion_r830683801) 
(I also express ~Strikethrough~ on the text. It it wrong information.)

In my opinion, we focus the wrong -L option in this issue.

Reply:
OK, I understand. Thanks !